src/test/java/org/testcontainers/containers/CartridgeContainerTestUtils.java

```diff
@@ -15,10 +15,10 @@ private CartridgeContainerTestUtils() {
 
     static public void executeProfileReplaceSmokeTest(TarantoolCartridgeContainer container) throws Exception {
         container.executeCommand(
-                "return profile_replace(...)", Arrays.asList(1, "Ivanov Ivan Ivanovich", 33, 100500)).get();
+                "return profile_replace({1, \"Ivanov Ivan Ivanovich\", 33, 100500})");
 
-        List<?> result = container.executeCommand("return profile_get(...)", 1).get();
+        ArrayList<?> result = container.executeCommandDecoded("return profile_get(1)");
```

Comment:
I don't get why we need a more specific type here. Do we use any `ArrayList` methods or properties? Is `ArrayList` always guaranteed as a result (as I see it is a generic parameter so it seems not)?